docker/setup-multiclusters-schema.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+
+set -x
```

Comment:
nit: set -e usually is worthwhile too, I realise this might be a copy from somewhere else, but lack of that flag is confusing usually